bingo/local_optimizers/continuous_local_opt.py

```diff
@@ -119,14 +127,22 @@ class ContinuousLocalOptimization(FitnessFunction):
         `fitness_function` must Be a valid `FitnessFunction` for the specified
         algorithm
     """
-    def __init__(self, fitness_function, algorithm='Nelder-Mead'):
+    def __init__(self, fitness_function, algorithm='Nelder-Mead',
+                 param_init_bounds=None, optimization_options=None):
```

Comment:
```suggestion
                 param_init_bounds=None, **optimization_options_kwargs):
```

could we do this as kwargs?

Reply:
yeah that makes sense to me, I changed __init__ and updated the test suite accordingly